cli/internal/cmd/create.go

```diff
@@ -34,215 +22,24 @@ func NewCreateCmd() *cobra.Command {
 		Short: "Create instances on a cloud platform for your Constellation cluster",
 		Long:  "Create instances on a cloud platform for your Constellation cluster.",
 		Args:  cobra.ExactArgs(0),
-		RunE:  runCreate,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.Flags().Bool("conformance", false, "")
+			cmd.Flags().Bool("skip-helm-wait", false, "")
+			cmd.Flags().Bool("merge-kubeconfig", false, "")
+			cmd.Flags().Duration("timeout", 5*time.Minute, "")
+			// Skip all phases but the infrastructure phase.
+			cmd.Flags().StringSlice("skip-phases", []string{
+				string(skipInitPhase), string(skipAttestationConfigPhase),
```

Comment:
was thinking if we should adjust `allPhases(except ...string)` and use here. this way we would be sure that we don't miss any phases if a new one gets added